pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -362,20 +357,31 @@ public boolean accept(File dir, String name) {
 
     ExecutorService executor = Executors.newFixedThreadPool(_numThreads);
     int cnt = 0;
-    for (final File file : files) {
+    for (final Path dataFilePath : dataFilePaths) {
       final int segCnt = cnt;
 
       executor.execute(new Runnable() {
         @Override
         public void run() {
           try {
             SegmentGeneratorConfig config = new SegmentGeneratorConfig(segmentGeneratorConfig);
-            config.setInputFilePath(file.getAbsolutePath());
+
```

Comment:
Just want to confirm that you've checked that this still works for create segment from local file system..?

Reply:
yes. I've also tested:
```
bin/pinot-admin.sh CreateSegment -dataDir /tmp/sample-pinot-data  -outDir /tmp/pinot-data -tableName sampleTable -segmentName sampleTable_0 -schemaFile /tmp/sample_schema.json -overwrite -format parquet
``